README.md

```diff
@@ -51,9 +51,8 @@ If you want to use full test-unit features:
 
 ## License
 
-(The Ruby License)
-
-This software is distributed under the same terms as ruby.
+This software is distributed under either the terms of
+[Ruby License](https://spdx.org/licenses/Ruby.html) or GPLv2.
```

Comment:
Is this URL suitable?
It refers https://www.ruby-lang.org/en/about/license.txt and it's the new Ruby license.
Why don't we refer our `COPYING` file?

Reply:
I thought it was the best URL -- the only one I found that doesn't contains BSD-2-Clause stuff.

> Why don't we refer our `COPYING` file?

Oh, I missed it. Let's go with it.